src/libs/actions/PaymentMethods.ts

```diff
@@ -28,45 +46,41 @@ function continueSetup() {
 }
 
 function openWalletPage() {
-    const onyxData = {
-        optimisticData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: true,
-            },
-        ],
-        successData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-        failureData: [
-            {
-                onyxMethod: Onyx.METHOD.MERGE,
-                key: ONYXKEYS.IS_LOADING_PAYMENT_METHODS,
-                value: false,
-            },
-        ],
-    };
-
-    return API.read('OpenPaymentsPage', {}, onyxData);
+    return API.read(
```

Comment:
@JKobrynski please create types for Api write, read according to this post https://callstack-hq.slack.com/archives/C05P72RDZD2/p1695813248659689

Reply:
Done!